tests/AssertTest.php

```diff
@@ -332,6 +332,8 @@ public function getTests()
             array('countBetween', array(array(0, 1, 2), 4, 5), false),
             array('countBetween', array(array(0, 1, 2), 1, 2), false),
             array('countBetween', array(array(0, 1, 2), 2, 5), true),
+            array('isList', array(array(1, 2, 3)), true),
+            array('isList', array(array('key' => 1, 'foo' => 2)), false),
```

Comment:
you should add a test with `array(0 => 1, 2 => 3)` too

Reply:
Ok, I'm adding it in. :+1: 
Any idea why are my current tests failing?